plugins/uefi/fu-plugin-uefi.c

```diff
@@ -337,6 +337,7 @@ fu_plugin_coldplug (FuPlugin *plugin, GError **error)
 		fu_device_add_guid (dev, "2d47f29b-83a2-4f31-a2e8-63474f4d4c2e");
 		fu_device_set_version (dev, "0");
 		fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_ALLOW_ONLINE);
+		fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_ALLOW_OFFLINE);
```

Comment:
really this shouldn't be offering online or offline, it should only be reboot.

Reply:
Specifically offering it for offline is going to cause 2 reboots before it's applied.  1 for rebooting and the systemd target picking it up, and then another for UEFI capsule to pick it up.